src/app/features/bitcoin-choose-fee/bitcoin-custom-fee/bitcoin-custom-fee.tsx

```diff
@@ -27,6 +33,9 @@ interface BitcoinCustomFeeProps {
   onValidateBitcoinSpend(value: number): boolean;
   recipient: string;
   setCustomFeeInitialValue: Dispatch<SetStateAction<string>>;
+  nativeSegwitSigner: number;
```

Comment:
What is this? `Signer` typically referrs to our `Signer` type that isn't used here.

Reply:
What should I use as **key** to store data?